webapp-django/crashstats/topcrashers/views.py

```diff
@@ -104,8 +105,19 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
-            # Number of startup crashes.
-            hit['startup_percent'] = 0
+            # Number of crashes happening during startup. This is defined by
+            # the software, as opposed to the next methid which relies on
```

Comment:
It's a crash annotation defined by Firefox: https://dxr.mozilla.org/mozilla-central/rev/dc89484d4b45abf442162e5ea2dd46f9de40197d/toolkit/xre/nsAppRunner.cpp#3240.